api/api/spec/spec.yaml

```diff
@@ -365,6 +365,22 @@ x-rbac-catalog:
         actions: ['rules:read']
         resources: ['rule:file:0610-win-ms_logs_rules.xml']
         effect: "allow"
+    'rules:update':
+      description: "Update or upload rule files to Wazuh manager"
+      resources:
+        - $ref: '#/x-rbac-catalog/resources/rule:file'
+      example:
+        actions: ['rules:update']
+        resources: ['rule:file:0610-win-ms_logs_rules.xml']
```

Comment:
I believe the update should be resourceless, can't assign resources to files not yet existent. We can discuss it.

Reply:
We discussed it and agreed to change it.